tests/integration/test_decision_worker.py

```diff
@@ -0,0 +1,147 @@
+"""
+Test worker process.
+"""
+
+import multiprocessing as mp
+import time
+
+from modules.decision import decision_worker
+from modules import drone_odometry_local
+from modules.cluster_estimation import cluster_estimation
+from modules import odometry_and_time
+from modules.decision import decision
+from utilities.workers import worker_controller
+from utilities.workers import queue_proxy_wrapper
+
+
+DISTANCE_SQUARED_THRESHOLD = 25  # squared meters
```

Comment:
Is this a unit of area?

Reply:
Yes.

Reply:
rename to reflect that it's a unit of area instead of the square of a 2D distance